.github/workflows/tools-test.yml

```diff
@@ -0,0 +1,31 @@
+name: tools-test
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
```

Comment:
Same here: it runs tests and is not building anything.

Reply:
[This example in the doc](https://docs.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#referencing-a-container-on-docker-hub) makes me think the name of the job may be arbitrary.